src/components/dialogs/restore-modification-dialog.tsx

```diff
@@ -98,15 +98,15 @@ const RestoreModificationDialog = ({ open, onClose, modifToRestore }: RestoreMod
 
     const getLabel = (modif: NetworkModificationMetadata) => {
         if (!modif) {
-            return null;
+            return '';
         }
         return intl.formatMessage(
             { id: 'network_modifications.' + modif.messageType },
             {
                 ...modif,
                 ...computeLabel(modif),
             }
-        );
+        ) as unknown as string;
```

Comment:
intl.formatMessage return a string so, I don't know why you have to use  **as unknown as string.**

Reply:
src/components/dialogs/restore-modification-dialog.tsx(134,21): error TS2322: Type '(modif: NetworkModificationMetadata) => string | null' is not assignable to type '(item: NetworkModificationMetadata) => string'.
  Type 'string | null' is not assignable to type 'string'.
    Type 'null' is not assignable to type 'string'.
Error: Process completed with exit code 2.I got this !

Reply:
For me since this code comes from the main, it's weird that the build crashes without this correction